src/Console/Command.php

```diff
@@ -249,4 +249,34 @@ public function abort($code = self::CODE_ERROR)
     {
         throw new StopException('Command aborted', $code);
     }
+
+    /**
+     * Execute another command with the provided set of arguments.
+     *
+     * @param string|\Cake\Console\Command $command The command class name or command instance.
+     * @param \Cake\Console\ConsoleIo $io The ConsoleIo instance to use for the executed command.
+     * @param array $args The arguments to invoke the command with.
+     * @return int|null The exit code or null for success of the command.
+     */
+    public function executeCommand($command, ConsoleIo $io, array $args = [])
```

Comment:
Can the method name and arguments order be harmonized with `CommandRunner::runCommand()`?
Could the `$io` argument be made optional and internally create a new instance if none provided?

Reply:
I could do that. Making a new instance will do bad things in tests which is why I made it a required argument.

Reply:
Done now :+1: